zuul.d/edpm_multinode.yaml

```diff
@@ -272,6 +272,10 @@
       cifmw_extras:
         - '@scenarios/centos-9/multinode-ci.yml'
         - '@scenarios/centos-9/hci_ceph_backends.yml'
+      cifmw_ceph_daemons_layout:
+        rgw_enabled: true
+        dashboard_enabled: true
+        cephfs_enabled: true
```

Comment:
So the idea here is, in the edpm_multinode job, is to test w/ rgw dashboard and cephfs in addition to RBD.

In the single node job though we won't have dashboard. Assuming the CI is OK with this extra resource usage we can get additional coverage.

I see the job is green now with this change so the asserts should have worked. Let's just link the logs showing the assert to be sure.